README.md

```diff
@@ -265,7 +265,7 @@ Inflector.ordinalize(-1021)  // => '-1021st'
 
 ```js
 Inflections inflections([string locale])
-inflections([string locale], [function(Inflections) fn])
+Inflections inflections([string locale], [function(Inflections) fn])
```

Comment:
Nothing is returned if a function is given as argument. See https://github.com/martinandert/inflected/blob/master/src/inflections.js#L14